submissions/Anatolii-Petrenko/html-css-popup/css/reset.css

```diff
@@ -0,0 +1,90 @@
+/* Reset Styles */
```

Comment:
Your PR should contain files for review, the file `reset.css` is not for review. remove it.

Reply:
Done.